libs/blocks/global-navigation/blocks/navDropdown/dropdown.js

```diff
@@ -0,0 +1,270 @@
+import { getAnalyticsValue, toFragment, decorateCta } from '../../utilities/utilities.js';
```

Comment:
can we rename the folder to dropdown or rename the files here to navDropdown.js & navDropdown.css?

Reply:
In my footer refactor work, I've moved the dropdown logic under a `utilities` folder and renamed it as 'menu'. This is because the footer uses the same logic to generate the menu, so it needed to be a bit more generic. Could this wait until we open the full footer PR?
<img width="243" alt="dropdown renamed to menu" src="https://user-images.githubusercontent.com/11267498/233390735-b252c0a8-fa81-4cd6-a6de-c10e2a94954d.png">

Reply:
yeah - sure 👍